src/declarations/wallet/index.js

```diff
@@ -4,8 +4,13 @@ import { Actor, HttpAgent } from "@dfinity/agent";
 import { idlFactory } from "./wallet.did.js";
 export { idlFactory } from "./wallet.did.js";
 
-// CANISTER_ID is replaced by webpack based on node environment
-export const canisterId = process.env.WALLET_CANISTER_ID;
+/* CANISTER_ID is replaced by webpack based on node environment
```

Comment:
I think you could remove this whole wallet folder

Reply:
We can have a separate PR for that